nomos-services/data-availability/network/src/lib.rs

```diff
@@ -63,6 +63,9 @@ use crate::{
 
 pub type DaAddressbook = AddressBook;
 
+// todo: export as config param when deployment config is merged
+const ASSIGNATIONS_PRUNE_WINDOW: u64 = 8;
```

Comment:
Lets just make it configurable and it will be good to go

Reply:
I am waiting for the config PR to be merged, but it was stale for a while and now I need to update it as Antonio merged another bunch of config PRs, as we have a special validator/executor case in DA which needs to be resolved and updated in that PR to accommodate new changes on master. I will try to do it today/tomorrow. 
@danielSanchezQ , @bacv Maybe we could merge this and I will include it in a config PR?